File: 2nd sem/assignment 4/p2.c
#include<stdio.h>
int* max(int *a,int *b,int *c);
void main()
{
	int a,b,c,*m;
	printf("Enter 3 number: ");
	scanf("%d%d%d",&a,&b,&c);
	m=max(&a,&b,&c);
	printf(" max = %d ",*m);
}
int* max(int *a,int *b,int *c)
{
	return (*a>*b && *a>*c)?a:(*b>*c)?b:c;
}

File: 3rd sem/IT Lab/assignment 3/p2.py
#2. Write a program to find whether a number is prime or not.
def pri(a):
    if(a<=1):
        return False
    for i in range(2,int(a**.5)+1):
        if(a%i==0):
            return False
    return True

try:
    a=int(input("ENter a no. "))
    print(f"{a} is {'not '*(not pri(a))}prime")
except(ValueError):
    print(" wrong input")
except:
    print(" some error")